packages/hadron-build/commands/release.js

```diff
@@ -366,6 +366,8 @@ const transformPackageJson = (CONFIG, done) => {
 const installDependencies = (CONFIG, done) => {
   var args = [
     'install',
+    '--registry https://registry.npmjs.org/',
+    '--legacy-peer-deps',
```

Comment:
Yeah, so this install is the one that causes issues during the build process. I was thinking over what would be the the better approach here and I think we should symlink the .npmrc from monorepo root to `packages/compass`, that way we will ensure that Compass will pick up correct options during the install, we don't have to keep unrelevant options hardcoded in hadron-build (what if I want to build compass package against a local registry? would not be possible with this change), and we make sure that we don't need to change options in too many places

Reply:
Yeah that's a much better idea!